modules/mobile-agents/android/client/client/src/main/java/org/wso2/emm/agent/services/LocalNotification.java

```diff
@@ -58,7 +58,7 @@ public static void startPolling(Context context) {
 	}
 
 	public static void stopPolling(Context context) {
-		if (localNoticicationInvoked == true) {
+		if (localNoticicationInvoked) {
 			localNoticicationInvoked = false;
 			Intent alarm = new Intent(context, AlarmReceiver.class);
 			PendingIntent sender = PendingIntent.getBroadcast(context, REQUEST_CODE, alarm, 0);
```

Comment:
What does "0" refer to? If this is some default value kinda thing, then you might have to define a constant and use upon it to be able to improve maintainability of the code.